coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/parsetree/AbstractASTVisitor.java

```diff
@@ -282,6 +282,17 @@ protected R visit(ASTNode node, C ctx) {
       case HiveParser.TOK_WINDOWVALUES:
         return visitWindowValues(node, ctx);
 
+      // See IdentifierParser.g:
+      case HiveParser.TOK_INTERVAL_DAY_LITERAL:
+      case HiveParser.TOK_INTERVAL_DAY_TIME_LITERAL:
+      case HiveParser.TOK_INTERVAL_HOUR_LITERAL:
+      case HiveParser.TOK_INTERVAL_MINUTE_LITERAL:
+      case HiveParser.TOK_INTERVAL_MONTH_LITERAL:
+      case HiveParser.TOK_INTERVAL_SECOND_LITERAL:
+      case HiveParser.TOK_INTERVAL_YEAR_LITERAL:
+      case HiveParser.TOK_INTERVAL_YEAR_MONTH_LITERAL:
```

Comment:
There are also `TOK_INTERVAL_DAY_TIME` and `TOK_INTERVAL_YEAR_MONTH`, do we need to add them?

Reply:
I did try to look for those but couldn't find how those are supported / generated.  Also Apache Calcite doesn't support DAY_TIME and YEAR_MONTH parsing yet.

Also I haven't seen a real use case for these two in practice.

So I guess it's OK to leave them out until a real use case comes up?

Reply:
Makes sense, thank you for explaining.